README.rst

```diff
@@ -178,62 +178,86 @@ GetComponentImages example:
     image = Image.open('/usr/src/tesseract/testing/phototest.tif')
     with PyTessBaseAPI() as api:
         api.SetImage(image)
-        boxes = api.GetComponentImages(RIL.TEXTLINE, True)
-        print('Found {} textline image components.'.format(len(boxes)))
-        for i, (im, box, _, _) in enumerate(boxes):
-            # im is a PIL image object
-            # box is a dict with x, y, w and h keys
-            api.SetRectangle(box['x'], box['y'], box['w'], box['h'])
-            ocrResult = api.GetUTF8Text()
-            conf = api.MeanTextConf()
-            print(u"Box[{0}]: x={x}, y={y}, w={w}, h={h}, "
-                  "confidence: {1}, text: {2}".format(i, conf, ocrResult, **box))
+        results = api.GetComponentImages(RIL.TEXTLINE, True)
+        print('Found {} textline image components.'.format(len(results)))
+        for i, (img, box, _, _) in enumerate(results):
+            # img is a PIL image object of the binarized line
+            # box is a dict with x, y, w and h keys (w.r.t. full image)
+            img.save('phototest_line{0}_{x}x{y}+{w}x{h}.png'.format(i, **box), format='PNG')
 
-Orientation and script detection (OSD):
-```````````````````````````````````````
```

Comment:
Unfortunately, the differ from here on compares the wrong segments. I did not replace the OSD example, but inserted a full GetIterator example (i.e. the second half of the above loop), and conflated the two OSD variants below into one.